frontend/src/app/components/tasque-board/tasque-board.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { faMagnifyingGlass, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { BoardModel } from '../../../core/models/board/board-model';
+
+@Component({
+  selector: 'tasque-board',
+  templateUrl: './tasque-board.component.html',
+  styleUrls: ['./tasque-board.component.sass']
+})
+export class TasqueBoardComponent implements OnInit {
+
+  public searchIcon = faMagnifyingGlass;
+  public plusIcon = faPlus;
+  public isCreateColumn: boolean;
+  public createColumnForm: FormGroup;
+  private newBoard: BoardModel;
+
+  public board: BoardModel[] = [
+    { columName: 'To Do', tasks: [] }, 
+    { columName: 'In Progress', tasks: [] }, 
+    { columName: 'Code Review', tasks: [] }, 
+    { columName: 'Done', tasks: [] },
+  ];
+
+  constructor(formBuilder: FormBuilder) { 
+    this.createColumnForm = formBuilder.group({
+      'columnName': ['', [Validators.required]]
+    });
+  }
+
+  ngOnInit(): void {
+  }
+
+  OpenAddColumn(): void {
```

Comment:
use either 'AddColumn' or 'CreateColumn' for all variables related to 'AddColumn' functionality

Reply:
Already, I use it everywhere ''add"